components/director/pkg/inputvalidation/validators_cert_subject_mappings.go

```diff
@@ -0,0 +1,130 @@
+package inputvalidation
+
+import (
+	"strings"
+
+	"github.com/kyma-incubator/compass/components/director/pkg/cert"
+	tenantEntity "github.com/kyma-incubator/compass/components/director/pkg/tenant"
+	"github.com/pkg/errors"
+)
+
+const (
+	// RuntimeType represents a runtime consumer type
+	RuntimeType = "Runtime"
+	// IntegrationSystemType represents a integration system consumer type
+	IntegrationSystemType = "Integration System"
+	// ApplicationType represents an application consumer type
+	ApplicationType = "Application"
+	// SuperAdminType represents a super admin consumer type, mainly used in local setup and E2E tests
+	SuperAdminType = "Super Admin"
+	// BusinessIntegrationType represents a business integration consumer type
+	BusinessIntegrationType = "Business Integration"
+	// TechnicalClient represents a technical client consumer type
+	TechnicalClient = "Technical Client"
+
+	// GlobalAccessLevel is an access level that is not tied to a specific tenant entity but rather it's used globally
+	GlobalAccessLevel = "global"
+)
+
+// SupportedConsumerTypes is a map of all supported consumer types
+var SupportedConsumerTypes = map[string]bool{
+	RuntimeType:             true,
+	IntegrationSystemType:   true,
+	ApplicationType:         true,
+	SuperAdminType:          true,
+	BusinessIntegrationType: true,
+	TechnicalClient:         true,
+}
+
+// SupportedAccessLevels is a map of all supported tenant access levels
+var SupportedAccessLevels = map[string]bool{
+	string(tenantEntity.Customer):      true,
```

Comment:
seems not(or at least from the IDE) but we actually need it because otherwise fails, I've tried it. The tenant access levels that we have are []strings bug the supportedAccessLevels are of type `Type`